middleware/routes/claims.js

```diff
@@ -154,29 +154,23 @@ router.post(
 );
 
 // Rename a single file in a claim.
-router.post(
```

Comment:
The new pre-commit linter and prettier tool reformatted this automatically?

Reply:
It shouldn't have if you didn't make any modifications to this file. Check this file out and don't touch it, ya jabroni.